float32 angle
int32 class_id
int32 octave
float32 xPos
float32 yPos
float32 response
float32 size